src/sentry/metrics/minimetrics.py

```diff
@@ -1,3 +1,5 @@
+# mypy: ignore-errors
+
 import random
```

Comment:
we shouldn't have conditional imports -- we pin an exact version of our dependencies

Reply:
There is no way around this. The module in question does not exist in the release version and this backend is only for when a sentry SDK is with an override in getsentry.